classes/Tools.php

```diff
@@ -1457,15 +1457,7 @@ public static function truncateString($text, $length = 120, $options = array())
 
 	public static function normalizeDirectory($directory)
 	{
-		$last = $directory[strlen($directory) - 1];
-
-		if (in_array($last, array('/', '\\')))
-		{
-			$directory[strlen($directory) - 1] = DIRECTORY_SEPARATOR;
-			return $directory;
-		}
-
-		$directory .= DIRECTORY_SEPARATOR;
+		$directory = $directory = rtrim($directory, '/\\').DIRECTORY_SEPARATOR;;
```

Comment:
Sorry, should I make a new separate commit or a new pull request in case of such a small correction?

> In classes/Tools.php:
> 
> > @@ -1457,15 +1457,7 @@ public static function truncateString($text, $length = 120, $options = array())
> >  
> >  	public static function normalizeDirectory($directory)
> >  	{
> > -		$last = $directory[strlen($directory) - 1];
> > -
> > -		if (in_array($last, array('/', '\\')))
> > -		{
> > -			$directory[strlen($directory) - 1] = DIRECTORY_SEPARATOR;
> > -			return $directory;
> > -		}
> > -
> > -		$directory .= DIRECTORY_SEPARATOR;
> > +		$directory = $directory = rtrim($directory, '/\\').DIRECTORY_SEPARATOR;;
> Hi,
> 
> Thank you for your PR, can you also remove double variable assignation ?
> 
> Best regards.
> 
> —
> Reply to this email directly or view it on GitHub.
>